_posts/2020-03-05-grayskull.md

```diff
@@ -0,0 +1,331 @@
+# By the power of Grayskull... I have the Conda recipe!
+
+*The main goal of the Skeleto~~n~~r is to conquer Grayskull.*
+
+## Introduction
+
+All jokes aside, the new project [grayskull :skull:](https://github.com/marcelotrevisani/grayskull) was created with the __intention__ of generating better Conda recipes that would allow to package properly projects available in different channels such as PyPI, CRAN, Conan, GitHub register, GitHub repositories and so on. On top of that, Grayskull is also being developed to help [conda-forge](https://conda-forge.org/) to update recipes.
+
+
+## Current status
+
+Currently, Grayskull (version ``0.1.9``) is able to generate recipes just looking for packages on [PyPI](https://pypi.org/), and it is available on [PyPI](https://pypi.org/project/grayskull/) and [conda-forge](https://github.com/conda-forge/grayskull-feedstock).
+The GitHub repository for this package is: [https://github.com/marcelotrevisani/grayskull](https://github.com/marcelotrevisani/grayskull)
+
+Before Grayskull, we basically just had ``conda-build skeleton`` to generate recipes for Python packages on PyPI. In all other aspects, the difference of quality of the generated recipes, and also the time spent to generate them have a big discrepancy when compared to``conda-build skeleton`` and ``grayskull``. Grayskull generates recipes taking in consideration the platform, Python version available, selectors, compilers (Fortran, C and C++), packages constrains, license type, license file, and so forth. It uses metadata available from multiple sources to try to create the best recipe possible.
+
+### Installation
+You can install ``grayskull`` using ``pip`` or ``conda``.
```

Comment:
Add a note mentioning that `grayskull` does not depend on `conda`, one can create recipes with minimum dependencies.

Reply:
Done!